WinUIGallery/ControlPages/DesignGuidance/TypographyPage.xaml

```diff
@@ -14,151 +14,218 @@
     x:Class="AppUIBasics.ControlPages.TypographyPage"
     xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:controls="using:WinUIGallery.DesktopWap.Controls"
     xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
     xmlns:local="using:AppUIBasics"
-    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" xmlns:controls="using:WinUIGallery.DesktopWap.Controls"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
     x:Name="compactPage"
     mc:Ignorable="d">
 
-    <StackPanel Orientation="Vertical">
+    <Grid>
```

Comment:
Spacing between header line and actual content seems a bit tight:

![Screenshot of Typographypage](https://user-images.githubusercontent.com/16122379/217064906-67b3997f-7b88-434f-aeb6-934e8117e6f0.png)